src/Symfony/Bridge/Doctrine/Id/UlidGenerator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Types;
```

Comment:
The namespace does not match the folder path. BTW, why not call it "Generator"?

Reply:
> why not call it "Generator"?

got it! 
yes, let's name it `Generator`
I'm looking forward to your PR :)
If you can add tests, that'd be awesome.

Reply:
`IdGenerator` actually.

Reply:
Sure, it was just not very clear to have the namespace "Id" but "Uid" looks better to me

Reply:
> The namespace does not match the folder path. BTW, why not call it "Generator"?

Good catch. 
I just haven't spent enough time with implementing those Generators.

Thx.